### PR TITLE
[BPK-3746] Bridge Snackbar on iOS

### PR DESCRIPTION
**Pull request by k0nserv**
+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-react-native/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: ios/NativeTests/RCTBPKSnackbarManagerTests.m [hunk 1]
```diff
@@ -0,0 +1,53 @@
+//
```
Comment (georgegillams) [inline ios/NativeTests/RCTBPKSnackbarManagerTests.m:1]: extraneous line

File: jest/setup.js [hunk 2]
```diff
@@ -34,3 +34,11 @@ NativeModules.AndroidBpkSnackbar = {
   show: jest.fn(),
   dismiss: jest.fn(),
 };
+
+NativeModules.BPKSnackbarManager = {
+  LENGTH_SHORT: 0,
```
Comment (georgegillams) [inline jest/setup.js:39]: What is this?

File: lib/bpk-component-snackbar/src/BpkSnackbar-test.ios.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/*
```
Comment (georgegillams) [inline lib/bpk-component-snackbar/src/BpkSnackbar-test.ios.js:1]: extraneous comment

File: jest/setup.js (hunk 2 above)
Comment (k0nserv) [reply jest/setup.js:39]: To mock these out during tests. It means we exercise all the code in `BpkSnackbar.js` and `NativeSnackbar.ios.js` but not the actual native implementation

File: UNRELEASED.md [hunk 4]
```diff
@@ -1,5 +1,16 @@
 # Unreleased
 
+**Breaking:**
+
+- bpk-component-snackbar
+  - Icon only actions now require an accessibility label(`accessibilityLabel`).
```
Comment (shaundon) [inline UNRELEASED.md:6]:
```suggestion
  - Icon-only actions now require an accessibility label (`accessibilityLabel`).
```

File: lib/ios/BackpackReactNative/Classes/SnackbarBridge/RCTBPKSnakbarManager.h [hunk 5]
```diff
@@ -0,0 +1,24 @@
+/*
```
Comment (shaundon) [inline lib/ios/BackpackReactNative/Classes/SnackbarBridge/RCTBPKSnakbarManager.h:1]: You misspelled 'snackbar' in this file name (also in the `.m` file)

Comment (k0nserv) [reply lib/ios/BackpackReactNative/Classes/SnackbarBridge/RCTBPKSnakbarManager.h:1]: Damnit

**Comment by k0nserv** [comment]
Fixed

**Pull request merged**